src/OMPLPlanner.cpp

```diff
@@ -344,6 +351,12 @@ OpenRAVE::PlannerStatus OMPLPlanner::PlanPath(OpenRAVE::TrajectoryBasePtr ptraj)
         // TODO: Configure anytime algorithms to keep planning.
         //m_simpleSetup->getGoal()->setMaximumPathLength(0.0);
 
+        // start validity checker
+        m_or_validity_checker->start();
+        BOOST_SCOPE_EXIT((m_or_validity_checker)) {
+            m_or_validity_checker->stop();
+        } BOOST_SCOPE_EXIT_END
+
```

Comment:
This is delicate -- what if the user has changed the environment between calls to `InitPlan` and `PlanPath`?  For example, what if they want to use different padding/models for validating starts/goals than for doing the actual planning (not sure if anyone actually does this, but they could).  I currently get around this by baking in each call to a function, since I can rely on the environment lock.